upgrading/rosa-hcp-upgrading.adoc

```diff
@@ -0,0 +1,29 @@
+:_content-type: ASSEMBLY
+[id="rosa-hcp-upgrading"]
+= Upgrading ROSA with hosted control planes clusters
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: rosa-hcp-upgrading
+
+toc::[]
+
+[id="rosa-hcp-lifecycle-policy_{context}"]
+== Life cycle policies and planning
+
+To plan an upgrade, review the xref:../rosa_architecture/rosa_policy_service_definition/rosa-life-cycle.adoc#rosa-life-cycle[{product-title} update life cycle]. The life cycle page includes release definitions, support and upgrade requirements, installation policy information and life cycle dates.
+
+Upgrades are manually initiated or automatically scheduled. Red Hat Site Reliability Engineers (SREs) monitor upgrade progress and remedy any issues encountered.
+
+[id="rosa-hcp-upgrading-a-cluster"]
+== Upgrading a ROSA cluster
+There are three methods to upgrade {{hcp-title-first} clusters:
+
+* Individual upgrades through the `rosa` CLI
+* Individual upgrades through the {cluster-manager-url} console
+* Recurring upgrades through the {cluster-manager-url} console
+
+[NOTE]
+====
+When following a scheduled upgrade policy, there might be a delay of an hour or more before the upgrade process begins, even if it is an immediate upgrade. Additionally, the duration of the upgrade might vary based on your workload configuration.
```

Comment:
I _believe_ it's less than an hour delay now. Need to confirm with Elad

Reply:
Also you can set a scheduled day/time which may be useful to mention

--schedule-date string             Next date the upgrade should run at the specified UTC time. Format should be 'yyyy-mm-dd'
      --schedule-time string             Next UTC time that the upgrade should run on the specified date. Format should be 'HH:mm'